### PR TITLE
[1password] Add new event.action to 1Password item usages

### PR DESCRIPTION
**Pull request by anthonythleung**
## What does this PR do?

This PR adds the `event.action` field to Item Usages stream and remove the newly added `location` field in favour of the built in GeoIP processor on 1Password Events Reporting integration package.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [x] I have added the new processor rule to the item usages and sign in attempts elasticsearch ingest pipeline
- [x] I have updated the tests to match the addition

## Related

#2367 

[1Password Events API Reference](https://support.1password.com/events-api-reference/#:~:text=A%20Client%20object.-,action,-string)

### CODE REVIEW

File: packages/1password/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.0"
+  changes:
+    - description: Add new "event.action" to item usage events
```
Comment (andrewkroh) [inline packages/1password/changelog.yml:4]:
```suggestion
    - description: Add new "event.action" to item_usages events.
```

That looks like the name being used by the data stream.

**Comment by andrewkroh** [comment]
There was a CI failure for:

> [2022-03-07T16:17:58.186Z] README.md is outdated. Rebuild the package with 'elastic-package build'
> [2022-03-07T16:17:58.186Z] Error: checking package failed: checking readme files are up-to-date failed: checking readme files are up-to-date failed

**Review by P1llus — APPROVED** [review]

**Comment by P1llus** [comment]
Much appreciated @anthonythleung!

I will merge this a bit later, and the new version should appear for users a bit after that.

**Pull request merged**